src/network/http_server.rs

```diff
@@ -1818,6 +1834,16 @@ fn map_proto_on_chain_event_to_json_on_chain_event(
                 expiry: body.expiry,
             });
         }
+        Some(on_chain_event::Body::TierPurchaseEventBody(body)) => {
```

Comment:
We should also update the user limits endpoint to return whether a user is pro or not. So we have a single place to get user status